auth/src/main/java/org/openedx/auth/presentation/signup/SignUpUIState.kt

```diff
@@ -6,6 +6,9 @@ import org.openedx.core.system.notifier.AppUpgradeEvent
 
 data class SignUpUIState(
     val allFields: List<RegistrationField> = emptyList(),
+    val requiredFields: List<RegistrationField> = emptyList(),
+    val optionalFields: List<RegistrationField> = emptyList(),
+    val agreementFields: List<RegistrationField> = emptyList(),
```

Comment:
same here.
```
val agreementText: String? = null,

Reply:
this code is a part of the OpenEdx project, so it should have a maximum compatibility with all OpenEdx instances, not only Edx